.github/workflows/make_release.yml

```diff
@@ -30,9 +30,47 @@ env:
   NPM_TAG: ""
 
 jobs:
+  package:
+    runs-on: ubuntu-latest
+    outputs:
+      hash: ${{ steps.hash.outputs.hash }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+      with:
+        fetch-depth: 0
+    - name: Prepare package
+      run: |
+        cargo package -p tfhe
+    - uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
+      with:
+        name: crate
+        path: target/package/*.crate
```

Comment:
I think that if we include the Cargo.lock file at the root of the repo we have a better shot at reproducibility, otherwise we'll try with this,